### ISSUE TITLE
cant import unbeliaveboat

### ISSUE THREAD

**Issue by moldender** [DiamondDemon669/unbelievaboat.py#1]
hi, so i've had muliple warning while opening the file and i cant import it, pycharm is indicating it red while blue color is working.
I tried pycharms built in fexes but nothing worked. 
Ty if you can reply

**Comment by DiamondDemon669** [comment]
did you misspell it like you did in the title?

**Comment by moldender** [comment]
yeah, i misspelled in the title but i literally copyd the title and checked if i can import it. But i think  i cannot import it because my python not satisfied with something

**Comment by moldender** [comment]
Oh, okay i somehow managed to import it, but when i try like get guild i get the following error:

TypeError: get_guild() missing 1 required positional argument: 'guild'


the line that i used: 

dpnze = Client.get_guild('788541612266750023')


T think the issue here is that i wrote 1 string and in this command it is needed to wrote a "self" or i dont really know. But when i try to like get user balance i write 2 strings and the first is detected as its putting a string into that self thing.
